structuredheadings/dialogs/selectstyle.js

```diff
@@ -1,13 +1,106 @@
+var selectedStyle;
+
+var columns = 4;
+var htmlContent = "";
+htmlContent += "<div>";
+htmlContent += "<table role=\"listbox\"";
+htmlContent += "style=\"width:100%;height:100%;border-collapse:separate;\"";
+htmlContent += "cellspacing=\"2\" cellpadding=\"2\"";
+if (CKEDITOR.env.ie && CKEDITOR.env.quirks) {
+  htmlContent += "style=\"position:absolute;\"";
+}
+htmlContent += "><tbody>";
```

Comment:
Yea, ultimately it is your call and what fits your plans.  However, if your eyes bled trying to stick the strings together, imagine coming back later and trying to edit the strings :p